### DISCUSSION TITLE
toast autohide in v5.3.1

### DISCUSSION THREAD

**Discussion by YohanWadia** [twbs/bootstrap#39115] [Q&A]
Hi guys im just trying to hide a toast automatically. I could do this with v4.6 with  **data-delay="2000"** ... how ever it doesnt work with this version... after reading on stack overflow this is my final code with autohide... but it still doesnt work.
I tried **data-autohide ... data-bs-autohide...** all the combinations.. nothing works.

```html
<div class="toast fade show" role="alert" aria-live="assertive" aria-atomic="true" data-bs-autohide="true" data-bs-delay="2000">
      <div class="toast-header">
        <svg class="bd-placeholder-img rounded me-2" width="20" height="20" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" preserveAspectRatio="xMidYMid slice" focusable="false"><rect width="100%" height="100%" fill="#007aff"></rect></svg>
        <strong class="me-auto">Bootstrap</strong>
        <small>11 mins ago</small>
        <button type="button" class="btn-close" data-bs-dismiss="toast" aria-label="Close"></button>
      </div>
      <div class="toast-body">
        Hello, world! This is a toast message.
      </div>
    </div>
```

**Comment by louismaximepiton** [comment]
Hey there, 
it seems to work well [here](https://codepen.io/louismaximepiton/pen/xxmGLQj). 
If this is not enough, it'd save some time to have a CodePen or StackBlitz example having the issue you describe.

**Reply by mxmissile** [reply]
Try with OP's html. No go.

**Comment by YohanWadia** [comment]
ok... am gona re try it... and will get back ... i'll post the code too if unsuccessful.
Ok.. one immediate thing i noticed... you dint put... 
**data-bs-autohide="true"** nor did you put **"fade show" in the class**
so, may be these 2 from my older 4.6v stopped it from happening? .. will check
